docs/plugins/data.malware-scan.md

```diff
@@ -0,0 +1,47 @@
+data.malware-scan
+==========
+
+This plugin extracts hashs(md5,sha1,sha256) of each attachment and checks they against MHR based blacklists such as 
+[MALWARE HASH REGISTRY][1].
+
+This plugin will reject any mail that have higher percentage equal to 25%.
+
+Configuration
+-------------
+
+This plugin reads configuration from data.malware-scan.ini.
+
+The main section defines global settings for all supported MHR Services are specified as INI section headings with the configuration for 
+each services within that section.
+
+The services section can contain the following options:
+
+* checkit
```

Comment:
`enabled` would be a far better name and would be more consistent with other plugins